.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{kt,kts}]
+ktlint_standard_function-naming = disabled
+#ktlint_function_naming_ignore_when_annotated_with=Composable
```

Comment:
Is this line needed? It doesn't seem to have caught any compose functions even with it commented out.

Reply:
Ah sorry, I was experimenting with the snapshot version of ktlint. I opened a bug to switch to this after ktlint 1.0.1 is released.

I'll delete this line for now.